packages/cli/src/credentials/credentials.service.ee.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable import/no-cycle */
+import { Db } from '..';
+import { CredentialsService } from './credentials.service';
+import { CredentialsEntity } from '../databases/entities/CredentialsEntity';
+import { SharedCredentials } from '../databases/entities/SharedCredentials';
+import { User } from '../databases/entities/User';
+import { RoleService } from '../role/role.service';
+
+export class EECreditentialsService extends CredentialsService {
+	static async isOwned(
+		user: User,
```

Comment:
yes, unfortunately we are very inconsistent whether the whole user should be provided or just an id. In general I would prefer to always pass the whole object around, because you simply have more information and also typeorm is build for that style.

e.g.
`SharedCredentials.findOne({ user })` vs. `SharedCredentials.findOne({ user: { id: userId } })`